src/libs/actions/App.ts

```diff
@@ -77,6 +78,38 @@ Onyx.connect({
     },
 });
 
+const KEYS_TO_PRESERVE: OnyxKey[] = [
+    ONYXKEYS.ACCOUNT,
```

Comment:
@marcaaron Was there any specific reason not to include `ONYXKEYS.USER` here? We're considering adding it in https://github.com/Expensify/App/issues/49087, but wanted to check first if there are any good arguments against doing so.

Reply:
Can't think of a reason! Thanks for asking! The array of keys also predates this PR so we could also ask the author of [the original PR](https://github.com/Expensify/App/pull/35306/files) that introduced this change.